src/mmw/apps/water_balance/templates/home_page/index.html

```diff
@@ -2,11 +2,15 @@
 
 {% load staticfiles %}
 
+{% block metatitle %}
+    <title>Model My Watershed - Micro Site Storm Model</title>
+{% endblock metatitle %}
+
 {% block header %}
     <header class="water-balance">
 
         <nav id="app-header" class="navbar-watershed">
-            <div class="brand"><a href="#">Model My Watershed</a></div>
+            <div class="brand"><a href="#">Model My Watershed - Micro Site Storm Model</a></div>
```

Comment:
There's a lot of styling associated with `.brand .a`, so removing the link on the second part changes that text, and I think it is harmless to have that noop link. Note that this subtitle is only available on the `/micro` site, so we are not polluting the base app with it.

Should the link to the main app open in a new tab? We don't persist the user state in the micro app anywhere, but there is still state that the user might not want to lose.